templates/user/settings/applications.tmpl

```diff
@@ -63,11 +69,11 @@
 		<p>{{.i18n.Tr "settings.access_token_deletion_desc"}}</p>
 	</div>
 	<div class="actions">
-		<div class="ui red basic inverted cancel button">
+		<div class="ui cancel button">
 			<i class="remove icon"></i>
 			{{.i18n.Tr "modal.no"}}
 		</div>
-		<div class="ui green basic inverted ok button">
+		<div class="ui red basic inverted ok button">
 			<i class="checkmark icon"></i>
 			{{.i18n.Tr "modal.yes"}}
```

Comment:
If we are improving UX already, wouldn't it be better to use `No -> Cancel` and `Yes -> Delete` instead?

Reply:
That would trigger a change in the translation. I wrote my thoughts in https://github.com/go-gitea/gitea/issues/3589#issuecomment-1146844860

Therefore, the PR only mentions, it is _associated_ with the issue, i.e. not a complete fix.

How long does it take to complete a translation cycle?
I would fill a follow-up PR to adjust the wording (as I mentioned, I would highlight that the change cannot be undone in this case). Inconsistency in modals are also discussed in https://github.com/go-gitea/gitea/issues/3605

I'm not sure, whether the Template should become a Vue component, for example. Or be made reusable.
I can see arguments pro and con.

Reply:
Given https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md#code-review 

> Make small pull requests. The smaller, the faster to review and the more likely it will be merged soon.

I will brew a follow-up PR with the wording changes.

Reply:
This PR is already incredibly small.
This is rather meant for bigger changes, i.e. PRs that change 400+ lines.